api/src/main/resources/rdr.yaml

```diff
@@ -0,0 +1,202 @@
+#This file provide the end points for interaction with RDR
+
+
+swagger: '2.0'
+
+info:
+  title: RDR
+  description: |
+    Service for RDR Interactions.
+  version: "0.1"
+#The host will get updated as per the environment it will be pmi-drc-api-test for dev and test
+host: "pmi-drc-api-test.appspot.com"
+schemes:
+  - "https"
+basePath: /rdr/v1
+
+securityDefinitions:
+  googleoauth:
+    type: oauth2
+    authorizationUrl: 'https://accounts.google.com/o/oauth2/auth'
+    flow: implicit
+    scopes:
+      openid: open id authorization
+      email: email authorization
+      profile: profile authorization
+
+security:
+  - googleoauth:
+    - openid
+    - email
+    - profile
+    - https://www.googleapis.com/auth/devstorage.full_control
```

Comment:
FWIW I got the error 

> Semantic error at security.0.3
> Security scope definition https://www.googleapis.com/auth/devstorage.full_control could not be resolved

 when validating this at https://editor.swagger.io/.